gap/grahom.gi

```diff
@@ -451,6 +451,50 @@ end);
 # IsDigraph{Homo/Epi/...}morphism
 ########################################################################
 
+# Given a non-negative integer <n> and a homogeneous list <partition>,
+# this global function tests whether <partition> is a valid partition
+# of the list [1 .. n]. A valid partition of [1 .. n] is either:
+#
+# 1. A list of length <n> consisting of a numbers, such that the set of these
+#    numbers is [1 .. m] for some m <= n.
+# 2. A list of non-empty disjoint lists whose union is [1 .. n].
+#
+# If <partition> is a valid partition of [1 .. n] then this global function
+# returns the partition, in form 1 (converting it to this form if necessary).
+# If <partition> is invalid, then the function returns <fail>.
+
+# Given:
+#
+# 1) two digraphs <src> and <ran>,
+# 2) a transformation or permutation <x> mapping the vertices of <src> to
+#    <ran>, and
+# 3) two lists <cols1> and <cols2> of positive integers defining vertex
+#    colourings of <src> and <ran>,
+#
+# this global function tests whether <x> respects the colouring, i.e. whether
+# for all vertices i in <src>, cols[i] = cols[i ^ x].
+
+InstallGlobalFunction(DIGRAPHS_RespectsColouring,
```

Comment:
Why not make this a documented operation?

Reply:
No reason, I realised that would be better a few minutes after pushing...